vignettes/sidebars.Rmd

```diff
@@ -58,215 +87,398 @@ iframe, .bslib-page-fill {
   border: var(--bs-border-width) solid var(--bs-border-color);
   margin-bottom: 4rem;
   box-shadow:
-    rgba(0, 0, 0, 0.25) 0px 54px 55px, 
-    rgba(0, 0, 0, 0.12) 0px -12px 30px, 
-    rgba(0, 0, 0, 0.12) 0px 4px 6px, 
-    rgba(0, 0, 0, 0.17) 0px 12px 13px, 
+    rgba(0, 0, 0, 0.25) 0px 54px 55px,
+    rgba(0, 0, 0, 0.12) 0px -12px 30px,
+    rgba(0, 0, 0, 0.12) 0px 4px 6px,
+    rgba(0, 0, 0, 0.17) 0px 12px 13px,
     rgba(0, 0, 0, 0.09) 0px -3px 5px;
 }
+
+iframe {
+  margin-bottom: calc(55px + 2rem);
+}
+
+iframe.resizable {
+  resize: vertical;
+  min-height: 300px;
+}
+
+.card-header.bg-info code {
+  background: none;
+  color: hsl(209deg, 85%, 25%);
+}
 ```
 
-### Sidebar layouts
+::: lead
+Sidebar layouts in web interfaces allow your users to easily access filters, settings and other inputs alongside the interactive features they control.
+In this article, you'll learn how to create sidebar layouts with bslib.
+:::
+
+## Introduction
+
+To create a sidebar layout, provide a `sidebar()` to the `sidebar` argument of one of the following functions.
+
+:::: {.row}
+::: {.col-sm-12 .col-lg-6}
+### Shared Sidebar {#intro-shared-sidebar}
+
+When `sidebar` is an argument to a function containing multiple pages or panels, it creates a **[shared sidebar](#shared-sidebar)**.
+
+Provide a `sidebar()` to `page_navbar()` to recreate the classic [shinydashboard](http://rstudio.github.io/shinydashboard/) look with a global sidebar [shared across several pages](#navbar-pages).
+
+You can also add a [shared sidebar to a card with tabs or pills](#card-tabs-shared-sidebar) created with `navs_tab_card()` or `navs_pill_card()`.
+:::
+
+::: {.col-sm-12 .col-lg-6 .text-center .mt-2 .mt-lg-4}
+```{r preview-page-navbar, echo = FALSE}
+describe_layout_function(
+  "page_navbar()",
+  "A shared sidebar in a multi-page layout.",
+  card_body_fillable(
+    class = "p-0",
+    height = 200,
+    id = "page_navbar_example_card",
+    navs_bar(
+      collapsible = FALSE,
+      fillable = TRUE,
+      sidebar = sidebar("Shared sidebar", width = "33%", class = "bg-light"),
+      nav("Page 1", "Page 1 content"),
+      nav("Page 2", "Page 2 content")
+    ),
+    tags$script(HTML("$(document).ready(function() {
+      var demo_nav = $('#page_navbar_example_card nav.navbar');
+      setTimeout(function(){ demo_nav.headroom('destroy') }, 100);
```

Comment:
Why is this needed? If it is, can we add a comment about why it is? (and also, maybe, check that headroom is on the page before using it since it's a pkgdown thing?)

Reply:
Oh right, yeah it's pkgdown thing where pkgdown enables headroom for anything that matches `nav.navbar`. That probably should be made a little more specific in pkgdown, but I'll add a comment here and check `.headroom` before calling `'destroy'`.